### PR TITLE
feat: add missing emojis to sentences that do not have them

### LINKED ISSUE #284: fix: Missing emojis
## Description
I noticed that some people forgot to add some emojis in the readme.

### PR DESCRIPTION
**Pull request by CBID2**
## Description
There was some sentences that were missing emojis, so I added them

## Issue
Closes #284

### CODE REVIEW

**Review by intelagense — CHANGES_REQUESTED** [review]
This is a fun PR! Although the instructions do say to add emojis to the end, I’m fine with bending the rules a bit to better accommodate a global audience. 

With that in mind, if a line already has an emoticon or emoji anywhere, let's keep it as is since it adds to the charm. If a line has neither, then adding an emoji would be great.

File: README.md [hunk 1]
```diff
@@ -303,17 +303,17 @@ Hi, markoagnes was here! This was added on 25-06-2024 🌙
 
 Hi, Shahzeb Ali was here! on 06/26/2024 🥸🥰
 
-Hi, my name is Marcellino Abdelmalak! This was added on 29-06-2024 (〃￣︶￣)人(￣︶￣〃)
+Hi, my name is Marcellino Abdelmalak! This was added on 29-06-2024 🌐
```
Comment (CBID2) [inline README.md:306]:
```suggestion
Hi, my name is Marcellino Abdelmalak! This was added on 29-06-2024 (〃￣︶￣)人(￣︶￣〃)
```

File: README.md [hunk 2]
```diff
@@ -115,25 +115,25 @@ Hi, my name is Osteduden! This was added on 18.03.2024 🦊
...
+Hi, my name is Xilient! This was added on <19-03-2024> 😎
 
 Hi, my name is Musa! This was added on 19-03-2024 🧸
 
 Hi, my name is Lui! This was added on 119-03-2024 👻
 
 Hi, Kvrae sends you greetings! This was added on 23/03/2024. 👀
 
-Hi, my name is Fredrik! This was added on 24-03-2024 :) Swag :)
+Hi, my name is Fredrik! This was added on 24-03-2024 😎
```
Comment (CBID2) [inline README.md:126]:
```suggestion
Hi, my name is Fredrik! This was added on 24-03-2024 :) Swag :)
```

File: README.md [hunk 3]
```diff
@@ -115,25 +115,25 @@ Hi, my name is Osteduden! This was added on 18.03.2024 🦊
...
 
-Hi, my name is Iamrushabhshahh! This was added on 05-03-2024 ❤️
+Hi, my name is Iamrushabhshahh! This was added on 05-03-2024  💛
```
Comment (CBID2) [inline README.md:128]:
```suggestion
Hi, my name is Iamrushabhshahh! This was added on 05-03-2024 ❤️
```

File: README.md [hunk 4]
```diff
@@ -143,13 +143,13 @@ Hi there, Paul is adding a contribution on 4/9/2024 😼
 
 Hi, my name is Renee! This was added on 07-04-2024 ☕
 
-Hi, my name is Berkay! This was added on 08.04.2024 ^-^
+Hi, my name is Berkay! This was added on 08.04.2024 🤗
```
Comment (CBID2) [inline README.md:146]:
```suggestion
Hi, my name is Berkay! This was added on 08.04.2024 ^-^
```

File: README.md [hunk 5]
```diff
@@ -217,15 +217,15 @@ Hi, my name is aresmendi! This was added on 23-05-2024 🎸
...
 
-Hi my name is Kunal Parkhade (KP)! This was added on 24/05/2024 💛IN
+Hi my name is Kunal Parkhade (KP)! This was added on 24/05/2024 💛
 
 Hi, my name is Trinity! This was added on 26-05-2024 👾
 
 Hi, my name is Hannah! This was added on 5/27/2024 🦆
 
-Hi, my name is Sérgio! This was added on 30/05/2024 :)
+Hi, my name is Sérgio! This was added on 30/05/2024  💛
```
Comment (CBID2) [inline README.md:226]:
```suggestion
Hi, my name is Sérgio! This was added on 30/05/2024  :)
```

File: README.md [hunk 6]
```diff
@@ -303,17 +303,17 @@ Hi, markoagnes was here! This was added on 25-06-2024 🌙
...
 Hi, Shahzeb Ali was here! on 06/26/2024 🥸🥰
 
-Hi, my name is Marcellino Abdelmalak! This was added on 29-06-2024 (〃￣︶￣)人(￣︶￣〃)
+Hi, my name is Marcellino Abdelmalak! This was added on 29-06-2024 🌐
 
-Hi from Russia! Nikita was here! 29/06/2024!
+Hi from Russia! Nikita was here! 29/06/2024! 😎
 
-Hi, my name is Lenny! This was added on 29-06-24 :)
+Hi, my name is Lenny! This was added on 29-06-24 🌙
```
Comment (CBID2) [inline README.md:310]:
```suggestion
Hi, my name is Lenny! This was added on 29-06-24 :)
```

File: README.md [hunk 7]
```diff
@@ -303,17 +303,17 @@ Hi, markoagnes was here! This was added on 25-06-2024 🌙
...
 
 Helloo! Brendon here. I added this on 06/29/2024 😁
 
 Hi, my name is Aaliyah! This was added on 30-06-2024 😊🌟💖
 
-Hey, Kshitij Akarsh was here on 01-06-2024. :)
+Hey, Kshitij Akarsh was here on 01-06-2024. 💖
```
Comment (CBID2) [inline README.md:316]:
```suggestion
Hey, Kshitij Akarsh was here on 01-06-2024. :)
```

**Review by CBID2 — COMMENTED** [review]
Add these changes

**Comment by CBID2** [comment]
> This is a fun PR! Although the instructions do say to add emojis to the end, I’m fine with bending the rules a bit to better accommodate a global audience.
> 
> With that in mind, if a line already has an emoticon or emoji anywhere, let's keep it as is since it adds to the charm. If a line has neither, then adding an emoji would be great.

Hi @intelagense! :) I made the changes

**Comment by CBID2** [comment]
Made the changes @intelagense

**Comment by intelagense** [comment]
I don't if I am looking at it wrong but I still see a few like ":)"s and "XD"s that have been altered in the PR. 🙏

File: README.md [hunk 8]
```diff
@@ -115,7 +115,7 @@ Hi, my name is Osteduden! This was added on 18.03.2024 🦊
 
 Hi, I (Vinay) was here too! 18-03-2024 🐒
 
-Hi, my name is Xilient! This was added on <19-03-2024> <:D>
+Hi, my name is Xilient! This was added on <19-03-2024> 😎
```
Comment (CBID2) [inline README.md:118]:
```suggestion
Hi, my name is Xilient! This was added on <19-03-2024> <:D>
```

File: README.md [hunk 9]
```diff
@@ -501,7 +501,7 @@ Hi, my name is Nassim! This was added on 12-10-2024 👌🖥️
 
 Hi, my name is Shahriar! This was added on 13-10-2024 😁
 
-HI! Jair Robles was Here🐨, 14th Oct 2024.
+HI! Jair Robles was Here🐨, 14th Oct 2024.  💛
```
Comment (CBID2) [inline README.md:504]:
```suggestion
HI! Jair Robles was Here🐨, 14th Oct 2024.  
```

File: README.md [hunk 10]
```diff
@@ -263,9 +263,9 @@ Hi, my name is Sophia! This was added on 10-06-2024 🥳
 
 Hi, my name is Alia! This was added on 10-06-2024 🫐🫐🫐
 
-Hi, my name is Trisheta! This was added on 11-06-2024 :)
+Hi, my name is Trisheta! This was added on 11-06-2024 ✅
 
-Hi, my name is Estarth! This was added on 11/06/2024 XD
+Hi, my name is Estarth! This was added on 11/06/2024 🍒
```
Comment (CBID2) [inline README.md:268]:
```suggestion
Hi, my name is Estarth! This was added on 11/06/2024 XD
```

**Review by CBID2 — COMMENTED** [review]
Make those changes

**Comment by CBID2** [comment]
> I don't if I am looking at it wrong but I still see a few like ":)"s and "XD"s that have been altered in the PR. 🙏 

Hey @intelagense! 😁 You're right. There were a few ":)"s and "XD"s that have been altered. I corrected them

**Review by intelagense — APPROVED** [review]

**Comment by intelagense** [comment]
I added this one back in and left the heart as well. "^\_^" I think it's maybe a kaomoji with a scar but I have no idea. 🤷 

Thanks for the PR!

**Comment by intelagense** [comment]
Ah the slash was the escape for the underscore in markdown. 👍

**Pull request merged**